easybuild/easyconfigs/v/VASP/VASP-5.4.1-intel-2016.02-GCC-4.9.eb

```diff
@@ -14,15 +14,32 @@ toolchainopts = {'usempi': True}
 # Vasp is proprietary software, see http://www.vasp.at/index.php/faqs on how to get access to the code
 sources = [SOURCELOWER_TAR_GZ]
 
+# uncommment after downloading patches from http://cms.mpi.univie.ac.at/wiki/index.php/Installing_VASP
+#patches = [
+#    'vasp.5.4.1.08072015.patch',
+#    'vasp.5.4.1.27082015.patch',
+#    'vasp.5.4.1.06112015.patch',
+#]
+
+checksums = [
+    '57c2b07d0f70987000033984e913f1a6',  # vasp.5.4.1.tar.gz
```

Comment:
@boegel at first I thought the problem was that there are two "5.4.1" versions, one from 24 June 2015 (the one I'm using) and another from 05 February 2016 (with GPU support). I had renamed the file to conform with `SOURCELOWER_TAR_GZ`, but I suppose it's better to keep the original filename to avoid confusion.

Still, you are right, if I download the "24Jun15" file now, it is different than the one I downloaded then (on 30 June 2015). Further inspection reveals that the only difference is that five files were removed from the `src` folder, on 19 September 2015, but the date was not updated neither in the filename nor in the site metadata...

I suppose it's much more likely that people will use a recently downloaded file than one downloaded during that three month period last year (and even if they have an old file and the checksum fails, the first thing they'd do is download it again), so I'll change the checksum to the new one.